login.html

```diff
@@ -440,11 +440,13 @@ <h4>Integrity Pledge
       </label>
      
  
-
+      <div class="col-12 col-sm-6 col-md-12">
```

Comment:
just remove all this boostrap styling. you can use media query with 768 px break points

Reply:
I tried using it for the footer-down class , it's not working.

Reply:
don't use it for `footer-down` class instead try it using `labelnew` class also first you should try your changes on less code pages like `help.html` Once succeed try to implement same in all pages.

Reply:
Ok I will let you know after implementing it  I guess both the code pages are almost same.

Reply:
@nlok5923  for labelnew class media query with 768 breakpoints is already implemented .Please check it once in the style.css file.

Reply:
yeah that's what i am saying try to manipulate position in that media query the styles mentioned in that media query are not correct and that's the point of this issue. As previously we had the toggle switch on top so may be the current styling in that media query is according to previous toggle button position (button in header) you need to fix that only via considering the newer changes (which is toggle button in footer).